tests/backup/backup_basic_test.go

```diff
@@ -230,174 +241,178 @@ var _ = AfterSuite(func() {
 	defer dash.TestSetEnd()
 	defer EndTorpedoTest()
 
-	ctx, err := backup.GetAdminCtxFromSecret()
-	log.FailOnError(err, "Fetching px-central-admin ctx")
-
-	//Cleanup policy
-	s3EncryptionPolicy := os.Getenv("S3_ENCRYPTION_POLICY")
-	if s3EncryptionPolicy != "" {
-		err = RemoveS3BucketPolicy(GlobalAWSBucketName)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Verify removal of S3 bucket policy"))
-	}
+	if envCleanup() {
```

Comment:
Add a log statement just before this line to indicate where cleanup will happen or not

Reply:
Done